vendor/enless-wireless/tx-amb-600-022.yaml

```diff
@@ -0,0 +1,104 @@
+name: TX VOC/T&H AMB 600-022
```

Comment:
Could you add a category to the name? So something along the lines of `TX VOC/T&H AMB 600-022 - Air Quality Sensor` (This is an example) 

Please do this for all devices as well.

Reply:
Ok will add the category name